SonarQube.MSBuild.Tasks/Targets/SonarQube.Integration.targets

```diff
@@ -433,12 +427,15 @@
     <PropertyGroup>
       <!-- Check whether to turn off code analysis logging -->
       <SonarQubeDisableRoslynCodeAnalysis Condition="$(SonarQubeExclude) == 'true' OR $(SonarQubeTestProject) == 'true' ">true</SonarQubeDisableRoslynCodeAnalysis>
-            
+
       <!-- Make sure no warnings are treated as errors -->
       <TreatWarningsAsErrors>false</TreatWarningsAsErrors>
       <WarningsAsErrors></WarningsAsErrors>
-    </PropertyGroup>
 
+      <!-- The Roslyn analyzers output messages with different warning levels. This is tied to the diagnostic severity and currently
+      is translated as follows - error: 0, warning: 1, info: 4. To avoid issues being missed override the warning level by setting it to max tolerance.-->
+      <WarningLevel>4</WarningLevel>
+    </PropertyGroup>
```

Comment:
This is the actual fix. The user might set this to another value so that he isn't bothered with Info messages - but this should be of little consequence in a lab build. However, there may be a perf impact if there are very many info messages. Still, from an analysis perspective, we want to capture all issues and let the quality profile filter them.